src/main/kotlin/org/pkl/lsp/ast/PklNode.kt

```diff
@@ -258,6 +260,16 @@ sealed interface PklProperty :
   val expr: PklExpr?
   val isDefinition: Boolean
   val typeAnnotation: PklTypeAnnotation?
+
+  fun toCompletionItem(): CompletionItem {
+    val item = CompletionItem(name)
+    item.kind = CompletionItemKind.Field
+    item.detail = type?.render() ?: "unknown"
+    if (this is PklClassProperty) {
+      item.documentation = getDoc(this, containingFile.pklProject)
```

Comment:
We can actually get rid of `effectiveDocComment`. I think I created it but forgot to get rid of it.

This can just be:

```suggestion
      item.documentation = parsedComment
```

Also, style-wise, I like using `apply` as a builder, e.g.

```
return CompletionItem(name).apply {
  kind = CompletionItemKind.Field
  // etc
}
```